crowdfunding/users/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+ path('', views.CustomUserList.as_view(), name='customuser-list'), #create and delete 
+ path('<int:pk>/', views.CustomUserDetail.as_view(), name='customuser-detail'),
+ path('change_password/<int:pk>/', views.ChangePasswordView.as_view(), name='change-password')
```

Comment:
i would have considered `/<int:pk>/change_password/` as the url, to keep it the same as the one above.